LoyaltyProgram_Intern/src/main/java/com/pgs/soft/service/UserService.java

```diff
@@ -9,11 +9,8 @@
 import com.pgs.soft.dto.UserDTO;
 
 @Service
-public interface UserService {
+public interface UserService extends GenericService<User, UserDTO, Integer> {
```

Comment:
I know, but  we already used Integer practically ewerywhere including database. Can it stay that way?